python/beeai_framework/backend/chat.py

```diff
@@ -56,7 +56,7 @@ class ChatConfig(BaseModel):
     parameters: ChatModelParameters | Callable[[ChatModelParameters], ChatModelParameters] | None = None
 
 
-class ChatModelStructureInput(ChatModelParameters):
+class ChatModelStructureInput(BaseModel):
```

Comment:
this appears to be reverting: https://github.com/i-am-bee/beeai-framework/pull/457/files#diff-08961e9238c2d441d0988309401935a99763661e09fe1a4a92fcec72e89ac6eaL59-R59